### ISSUE TITLE
How do i update VitaDeploy running on vs0?

### ISSUE THREAD

**Issue by InsanePrimate** [SKGleba/VitaDeploy#37]
I noticed VitaDeploy was updated from 1.2 to 1.21, and I was wondering ho I would go about updating VitaDeploy running on vs0. I tried simply replacing "near" again, but it just gave me an error, "already running on vs0"

**Comment by SKGleba** [comment]
Yeah you would need to remove and replace again, or re-replace via henlo_jb.
I will add an updater in the next ver
